package com.kodilla.exception.test;

public class SecondChallengeExeption extends Exception{

}
